include/mav/Network.h

```diff
@@ -404,6 +411,18 @@ namespace mav {
             _heartbeat_message = std::nullopt;
         }
 
+        void setGetTimestampFunction(std::function<uint64_t(void)> function) {
+            _get_timestamp_function = function;
+        }
+
+        void setKey(std::array<uint8_t, 32> key) {
+            _key = key;
+        }
+
+        void sendMessage(Message &message) {
+            _sendMessage(message, {});
```

Comment:
What's this for?
Generally, the NetworkInterface shouldn't be the interface for sending Messages, the Connection Object should be it. Directly sending messages on the Network Interface with default constructed ConnectionPartner would mean broadcasting a message to all known clients - a feature we probably don't want in the public API as it is an anti-pattern except for HEARTBEAT.

Reply:
Sorry, I forgot about this change. This should really be a separate PR as its unrelated to packet signing. We have a reason for it, but I'll revert it here and explain in a separate PR.

Reply:
See #34